helm/nifikop/templates/deployment.yaml

```diff
@@ -66,14 +66,18 @@ spec:
         - command:
             - /manager
           args:
-            - metrics-bind-address={{.Values.metrics.port}}
+            - metrics-bind-address={{ .Values.metrics.port }}
             - --leader-elect
             {{- if .Values.certManager.enabled }}
             - --cert-manager-enabled={{ .Values.certManager.enabled }}
             {{- end }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: "{{ .Values.image.pullPolicy }}"
           name: {{ template "nifikop.name" . }}
+          ports:
+            - name: metrics
+              containerPort: {{ .Values.metrics.port }}
```

Comment:
i think this would now work since the main.go _I think_ at this time has hardcoded metric port on 8080. I am not a go developer 😄

Reply:
I believe `:8080` is just the default metrics port: https://github.com/Orange-OpenSource/nifikop/blob/master/main.go#L68

Adding the flag to the `deployment` yaml overrides it: https://github.com/Orange-OpenSource/nifikop/blob/master/helm/nifikop/templates/deployment.yaml#L69